pyriksdagen/utils.py

```diff
@@ -26,6 +26,10 @@
 XML_NS = "{http://www.w3.org/XML/1998/namespace}"
 TEI_NS = "{http://www.tei-c.org/ns/1.0}"
 
+def fetch_ns():
+    return {"tei_ns": "{http://www.tei-c.org/ns/1.0}",
```

Comment:
Use the lines above?

Reply:
I use namespaces as a dict object all over the project. Would you be happy with a compromise?

```
def fetch_ns():
    return {"tei_ns": TEI_NS,
            "xml_ns": XML_NS}
```

Reply:
![image](https://github.com/user-attachments/assets/2c269363-e9a9-4464-bfee-8d88b64b7b9d)
![image](https://github.com/user-attachments/assets/a461462c-9acf-4276-9537-e681cf7d608b)

Reply:
https://github.com/swerik-project/pyriksdagen/pull/15/commits/eaf1f83c3ed935c7cd77535abca123abdfcb3348